crates/zed/src/languages/java/brackets.scm

```diff
@@ -0,0 +1,5 @@
+("(" @open ")" @close)
+("[" @open "]" @close)
+("{" @open "}" @close)
+("<" @open ">" @close)
+("\"" @open "\"" @close)
```

Comment:
I would add single quotes too. I know it's less important than double quotes because you should only ever be quoting a single character, but in my opinion it is still useful. I have the same opinion about the Rust support in Zed. Selecting a single character and hitting `'` should wrap the character in `'` instead of replacing it. If it's not feasible or desired to special-case that, then I would be happy if it just supported wrapping any number of characters instead, like with double quotes.

Reply:
Sure, will add it, once I am back to java support